### PR TITLE
remove presence list tests

### PR DESCRIPTION
**Pull request by neilisfragile**

### CODE REVIEW

File: tests/21presence-events.pl [hunk 1]
```diff
@@ -1,108 +0,0 @@
-# Eventually this will be changed; see SPEC-53
-my $PRESENCE_LIST_URI = "/r0/presence/list/:user_id";
-
-
-test "initialSync sees my presence status",
```
Comment (richvdh) [inline tests/21presence-events.pl]: this doesn't seem to be related to presence lists?

Comment (neilisfragile) [reply tests/21presence-events.pl]: yup saw the same - sec

**Comment by richvdh** [comment]
@neilisfragile did you mean to close this?

**Comment by richvdh** [comment]
oh, seems to be superceded by #600
